certify-default.properties

```diff
@@ -144,3 +146,10 @@ mosip.certify.domain.url=
 mosip.certify.key-values={}
 mosip.certify.database.name=inji_certify
 mosip.certify.authorization.url=
+# To specify whether plugin or certify should issue a VC. Supported values are "PluginIssuer" and "CertifyIssuer"
+mosip.certify.issuer=PluginIssuer
```

Comment:
This can't be kept empty @vishwa-vyom. This enables VCIssuance type of plugins, such that no config change is required in the config files of mosipid, mock, insurance/sunbird. But this value is overridden in the DataProvider plugin implemented usecases to `CertifyIssuer`.

Reply:
Keepting this empty and updating the other properties will make it clear to anyone implementation a new plugin has to override this property in their property specific file. Keep with PluginIssuer is also ok, but there is a slight inconsistency in this section.

Reply:
Property set as empty in certify-default and added with PluginIssuer value in mock, mosipid and sunbird properties

Reply:
this is great, one more small change.

Lets add this under "Plugin enable properties " section.
Also mosip.certify.integration.data-provider-plugin to be moved to "mosip.certify.integration.data-provider-plugin" section